### PR TITLE
Feature import wrapper function for dnpImport

### PR DESCRIPTION
**Pull request by tkeller12**

### CODE REVIEW

File: dnplab/__init__.py [hunk 1]
```diff
@@ -7,4 +7,5 @@
 from . import hydrationGUI
 from . import core
 from . import dnpResults
+from .dnpIO import import_data
```
Comment (ylin00) [inline dnplab/__init__.py]: How about `load_data`, `load` per easyspin style, or `read_data` or `read` per `nmrglue.bruker.read`, `pandas.read_XXX` ?

File: dnplab/dnpIO.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    elif data_type == "topspin":
+        return dnpImport.topspin.import_topspin(path, *args, **kwargs)
+
+    elif data_type == "topspin dir":
+        return dnpImport.topspin.import_topspin_dir(path, *args, **kwargs)
+
+    elif data_type == "vnmrj":
+        return dnpImport.vnmrj.import_vnmrj(path, *args, **kwargs)
+
```
Comment (ylin00) [inline dnplab/dnpIO.py]: missing `else`

File: dnplab/dnpIO.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        data_type (str): Type of data to import
+
+    Returns:
+        data (dnpData): Data object
+    """
+
+    if data_type == None:
+        data_type = autodetect(path)
+
+    if data_type == NotImplemented:
```
Comment (ylin00) [inline dnplab/dnpIO.py]: NotImplemented data type should raise an Exception?

File: dnplab/dnpIO.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+from . import dnpImport
+
+
+def import_data(path, data_type=None, *args, **kwargs):
+    """Import data from different spectrometer formats
+
+    Args:
+        path (str): Path to data directory or file
+        data_type (str): Type of data to import
```
Comment (ylin00) [inline dnplab/dnpIO.py]: How about `manufacturer`? or something else more **specific** and **explicit**

Comment (ylin00) [reply dnplab/dnpIO.py]: What are the list of possible options?

File: unittests/test_import_wrapper.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class import_data_tester(unittest.TestCase):
+    def setUp(self):
+        self.topspin_dir = "./data/topspin/"
+        self.prospa_dir = "./data/prospa/toluene_10mM_Tempone"
+        self.vnmrj_dir = "./data/vnmrj/10mM_tempol_in_water_array.fid"
+
+    def test_topspin(self):
+        data = import_data(self.topspin_dir, "topspin", 1)  # last arg is expNum
```
Comment (ylin00) [inline unittests/test_import_wrapper.py]: `expNum` should be a keyword argument, instead of positional argument, to be explicit and specific.

**Review by ylin00 — CHANGES_REQUESTED** [review]

File: dnplab/__init__.py (hunk 1 above)
Comment (tkeller12) [reply dnplab/__init__.py]: I like calling the wrapper function "load".

File: dnplab/dnpIO.py (hunk 2 above)
Comment (tkeller12) [reply dnplab/dnpIO.py]: Added `else`

**Review by ylin00 — APPROVED** [review]

**Review by thcasey3 — CHANGES_REQUESTED** [review]
@tkeller12 @ylin00 It seems to me that the way this is currently arranged the user interface is dnpIO and this calls dnpImport in the background, shouldn't it be the other way around? It will make more sense to a user to use dnpImport than dnpIO. Also, weren't we discussing changing to "load" or "read" rather than "import"? I vote for dnpLoad

File: unittests/test_import_wrapper.py (hunk 5 above)
Comment (tkeller12) [reply unittests/test_import_wrapper.py]: Fixed this

**Pull request merged**